index.html

```diff
@@ -304,63 +303,63 @@ <h2>Use Cases</h2>
           <dt>Render intensive application:</dt><dd>
               <p>
                 Render intensive applications are client side applications whose main task is to fetch content from the backend server then render that content in the front-end.
-                For example, news, social media Web applications, and mobile applications belongs to this category.
+                For example, news, social media Web applications, and mobile applications belong to this category.
               </p>
         </dd>
         <dt>Computing intensive application:</dt><dd>
               <p>
-                Computing intensive applications are client side applications whose main task is to do computing intensive work on the client. For example, mobile gaming applications 
-                need to calculate certain objects' location and other complex parameters based on user interaction then rendering in the client side. 
+                Computing intensive applications are client side applications whose main task is to do computing intensive work on the client. For example, mobile gaming applications
+                need to calculate certain objects' location and other complex parameters based on user interaction then rendering in the client side.
               </p>
         </dd>
         <dt>Hybrid application:</dt><dd>
               <p>
-                Hybrid applications are applications whose main task includes both rendering intensive work and computing intensive work. For example, modern e-commerce mobile applications 
+                Hybrid applications are applications whose main task includes both rendering intensive work and computing intensive work. For example, modern e-commerce mobile applications
                 leverage machine learning inference in the client side for AR/VR type user experiences. At the same time, e-commerce mobile applications need to fetch dynamic content based
                 on user preferences.
               </p>
         </dd>
         <dt>Mobile/static client:</dt><dd>
             <p>
-              Some client side applications remain static most of the time. For example, a camera for traffic monitoring and analysis does not require mobility support. 
-              On the other hand, some client side applications will change location continuously. For example, applications running on a connected vehicle or self driving vehicle 
+              Some client side applications remain static most of the time. For example, a camera for traffic monitoring and analysis does not require mobility support.
```

Comment:
Use of "static" here refers to location, not other forms of static (e.g. measurements not changing much).
```suggestion
              Some client side applications remain at a static location most of the time. For example, a camera for traffic monitoring and analysis does not require mobility support.
```